client/checkout/upe-styles/index.js

```diff
@@ -348,12 +362,31 @@ export const getAppearance = ( isBlocksCheckout = false ) => {
 		color: selectedTabRules.color,
 	};
 
+	const blockRules = getFieldStyles(
+		selectors.upeThemeLabelSelector,
+		'.Block'
+	);
+	blockRules.backgroundColor = 'none';
+
+	const globalRules = {
+		colorBackground: 'none',
```

Comment:
These fields will actually throw a warning in the console stating the following...

> [Stripe.js] stripe.elements(): invalid variable value "none" provided to "colorBackground"; "colorBackground" accepts a valid HEX, rgb(), or hsl() CSS color.

...but despite this it still works and I cannot figure out another way to tell Stripe that I don't want any background colour.

If you set any colour to this value, then payment elements for methods like Affirm and giropay will have a coloured box around them, which we do not want. You cannot set a transparent colour and my own function, `getBackgroundColor`, is not reliable enough to _always_ find the exact colour background for the payment method container. 

Ignoring this parameter entirely will also incur a coloured box background, since Stripe will use its default value instead, which looks particularly unattractive on the `night` theme.

![Block background colour box](https://cdn-std.droplr.net/files/acc_1104206/Vcg0kL)
_See this stupid faint grey box that I never want to see again_